wiki/Spotlight Guide.md

```diff
@@ -0,0 +1,142 @@
+Feature Spotlights
+# Introduction
+
+Spotlighting is a visual tool that highlights and brings a user’s focus to an element on the screen. We use them to communicate the purpose of a specific screen element to the user, which they might otherwise miss. Spotlighting involves dimming the other areas on the screen and lighting the element to bring the user’s focus onto it, and explaining what/how that feature should be used. 
```

Comment:
At the start: "Spotlights are a visual tool that ..."

Maybe, below this paragraph, add a new one that's like: "Here is an example of a spotlight in the Home screen:" (or wherever) and add a picture to show what a spotlight looks like in practice.

Reply:
Done - I'm wondering if there's a way to make this image look smaller?

Reply:
Probably the easiest is to use a smaller image? Feel free to do that in a separate PR though.